packages/mantine/src/styles/RowLayout.module.css

```diff
@@ -42,33 +42,38 @@
             color: var(--mantine-color-gray-5);
         }
     }
-}
-.rowCollapsibleButtonCell {
-    text-align: right;
-    padding: calc(var(--mantine-spacing-xs) / 2) var(--mantine-spacing-sm) !important;
-}
-.row {
-    & td:first-of-type {
-        padding-left: 40px;
-    }
-    @mixin hover {
-        @mixin light {
-            background-color: var(--mantine-color-gray-1);
+    &[data-selected='true'] {
+        &[data-multi-selection='false'] {
+            @mixin light {
+                background-color: var(--mantine-color-gray-1);
+            }
+            @mixin dark {
+                background-color: rgba(var(--mantine-color-primary-7), 0.2);
+            }
         }
-        @mixin dark {
-            background-color: rgba(var(--mantine-color-primary-7), 0.2);
+        &[data-multi-selection='true'] {
+            background-color: transparent;
         }
     }
-    overflow-wrap: anywhere;
+
+    &[data-collapsible-cell='true'] {
+        text-align: right;
+        padding: calc(var(--mantine-spacing-xs) / 2) var(--mantine-spacing-sm) !important;
+    }
 }
+
 .cell {
     vertical-align: middle;
     /* We must use height instead of minHeight here, otherwise it doesn’t apply*/
     height: 56px;
     padding: var(--mantine-spacing-xs) var(--mantine-spacing-sm);
     border-bottom: rem(1) solid var(--mantine-color-gray-3);
 }
-.collapsible {
+
+.collapsibleRow {
+}
```

Comment:
Do we actually HAVE to define those in our module for them to work when using `styles` ?

Reply:
We would need to try it, I'm not sure 🤔

Reply:
I tried it and since it's in css directly we can omit those empty rules (we couldn't when it was css in js). I made a cleanup across all css module files